plugins/mac-virtualcam/src/camera-extension/cmake/macos/Info.plist.in

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>CFBundleName</key>
+    <string>com.obsproject.obs-studio.mac-camera-extension</string>
+    <key>CFBundleIdentifier</key>
+    <string>com.obsproject.obs-studio.mac-camera-extension</string>
+    <key>CFBundleVersion</key>
+    <string>${MACOSX_BUNDLE_BUNDLE_VERSION}</string>
+    <key>CFBundleShortVersionString</key>
+    <string>${MACOSX_BUNDLE_SHORT_VERSION_STRING}</string>
+    <key>CFBundleExecutable</key>
+    <string>com.obsproject.obs-studio.mac-camera-extension</string>
+    <key>CFBundleInfoDictionaryVersion</key>
+    <string>6.0</string>
+    <key>CFBundlePackageType</key>
+    <string>SYSX</string>
+    <key>CFBundleSupportedPlatforms</key>
+    <array>
+        <string>MacOSX</string>
+    </array>
+    <key>LSMinimumSystemVersion</key>
+    <string>13.0</string>
+    <key>NSHumanReadableCopyright</key>
+    <string>(c) 2012-${CURRENT_YEAR} Hugh Bailey</string>
```

Comment:
Crediting Jim with the copyright on this code is certainly nice to him but not correct I think, given that it's the you and me who wrote it. Maybe something more generic like "OBS Project" would be more correct.